### PR TITLE
Fix receive your benefits link and add text slugs for links

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR changes the receive your benefits link to the correct tab and adds text slugs for the navbar, buttons, and other links, so that we can label them accordingly in analytics

### CODE REVIEW

File: src/client/components/TabPaneContent0/index.js [hunk 1]
```diff
@@ -38,16 +51,8 @@ function TabPaneContent0({ loadTab }) {
...
       </ul>
       <p>
-        If you're already receiving UI, review{" "}
-        <a
-          href="#tab0Title"
-          onClick={() => {
-            loadTab(1);
-          }}
-        >
-          Receive Your Benefits
```
Comment (sharonwarner) [inline src/client/components/TabPaneContent0/index.js:48]: Oh phew! I just noticed this was broken!

**Comment by idealisms** [comment]
I'm seeing titles and tabs showing up as "tabTitles.0". Do we need to make the indexes in the translation files start with a letter?

**Comment by kalvinwang** [comment]
@idealisms they shouldn't, i18n seems to support keys starting with numbers. try hard-refreshing / shift-refreshing?

**Comment by sharonwarner** [comment]
hard-refreshing updated the titles for me!

**Comment by kalvinwang** [comment]
@sharonwarner fixed the initial scroll issue!

**Pull request merged**